firestore-send-email/functions/__tests__/helpers.test.ts

```diff
@@ -0,0 +1,53 @@
+import Mail = require("nodemailer/lib/mailer");
+import { setSmtpCredentials } from "../src/helpers";
+import { Config } from "../src/types";
+
+describe("function setSmtpCredential test returning creating nodemailer transport based on ENV config", () => {
+  test("Mail config without smtpConnectionUri(SMTP_CONNECTION_URI env variable), old setting", () => {
+    const config: Config = {
+      smtpServerDomain: "smtp.gmail.com:25",
+      location: "",
+      mailCollection: "",
+      defaultFrom: "",
+    };
+    const credentials = setSmtpCredentials(config);
+    expect(credentials).toBeInstanceOf(Mail);
```

Comment:
Can we add additional tests on the configuration. For example: checking that the credentials has the correct domain name / port etc.

Reply:
Sure : )